### DISCUSSION TITLE
Query regarding some courses that do not fulfill MS requirements

### DISCUSSION THREAD

**Discussion by SiddharthChillale** [snigi-gupta/AttendingUniversityAtBuffalo#37] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/42491188/121906430-bef50480-cd48-11eb-944c-0963955b8131.png)
* Referring to the image, it says that the course **CSE 580** (Computer Graphics) _does not fulfill either of MS breadth or depth requirements_. I really want to take this course as I want to study in the computer graphics domain. However, I'm not sure if this course will be counted in the MS requirements or not. 
* Looking more into this, I found that courses **CSE 526** (Blockchain App development), **CSE 528** (Intro to image processing), **CSE 610** (Special Topics) also have the same message (as given in image that they do not fulfill MS course requiremets ). What's up with these particular courses ? They do not fulfill MS depth or breadth requirement but are still given in the graduate academic schedule ?
* Another side question - would you (or any of your friends) have any knowledge on CSE 580 (Computer Graphics), either the course or the professor (JingJing Meng) ?
*  Any help would be appreciated. Thanks for the amazing repo !

**Comment by snigi-gupta** [comment, marked as answer]
Hi @SiddharthChillale,

Firstly I sincerely apologize for not answering your questions. I was moving cities and hence could not give time to this repo. 
I am not sure if my answer will still help you or not, but I would still like to answer it.

CSE580 is an elective. For your MS requirement, you will be using only 24 credits. The rest 6 credits are basically for courses like these where you can study what you want. If you are opting for courses that are going to get you 24 credits according to the MS requirements, then you can choose this course, if you really want to study it. Basically, you can study any course you want upto 6 credits.

I think I answered your second point above. All these courses come into the electives category. Sometimes, these courses also become MS requirement-friendly. The best bet is to email the professor and ask if this course will satisfy any breadth/depth/project requirements.

I don't have any friends who opted CSE580. Maybe you can check ratemyprofessor.com. I am sorry, I don't think so I will be able to be of much help here. Try to audit the course if possible for you.

Feel free to post more questions if you have them. I am answering these every Sunday now. :)

**Comment by SiddharthChillale** [comment]
Thanks for replying. Your answer is still helpful. I did email the professor but didn't receive a reply yet. It seems the only option is to audit the course.
